### PR TITLE
Add links to resources not included in RTD

### PR DESCRIPTION
**Pull request by vojtechsokol**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
-   `Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework
+=============================================
+`Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework.
+
+Resources not included in this RTD:
+ - RPM dependencies for `leapp-repository <https://github.com/oamg/leapp-repository/blob/master/docs/dependencies.md>`_.
+ - Upgrade inhibition by producing a `inhibiting report message <https://github.com/oamg/leapp-repository/blob/master/docs/howto-inhibit-upgrade.md>`_.
+ - Type of tests used for `testing the actors <https://github.com/oamg/leapp-repository/blob/master/docs/tests.md>`_.
+ - NVR schema used for for leapp-related `rpm packages <https://github.com/oamg/leapp-guidelines/blob/master/build-schema.md>`_.
+ - Overview (slightly outdated) of phases used in `Inplace upgrades Workflow <https://raw.githubusercontent.com/leapp-to/leapp-to.github.io/master/img/phases.png>`_.
```
Comment (jmikovic) [inline docs/source/index.rst]: I wouldn't include the outdated list of phases

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,6 +1,15 @@
 Welcome to developer documentation for Leapp!
-=================================
-   `Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework
+=============================================
+`Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework.
+
+Resources not included in this RTD:
+ - RPM dependencies for `leapp-repository <https://github.com/oamg/leapp-repository/blob/master/docs/dependencies.md>`_.
+ - Upgrade inhibition by producing a `inhibiting report message <https://github.com/oamg/leapp-repository/blob/master/docs/howto-inhibit-upgrade.md>`_.
```
Comment (jmikovic) [inline docs/source/index.rst]: This should be turned into proper tutorial

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+ - Type of tests used for `testing the actors <https://github.com/oamg/leapp-repository/blob/master/docs/tests.md>`_.
+ - NVR schema used for for leapp-related `rpm packages <https://github.com/oamg/leapp-guidelines/blob/master/build-schema.md>`_.
+ - Overview (slightly outdated) of phases used in `Inplace upgrades Workflow <https://raw.githubusercontent.com/leapp-to/leapp-to.github.io/master/img/phases.png>`_.
+   For up-to-date state, consult the `source of IPU workflow <https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/workflows/inplace_upgrade.py>`_.
```
Comment (jmikovic) [inline docs/source/index.rst]: I think this is already included in one of the tutorials

File: docs/source/index.rst (hunk 2 above)
Comment (examon) [reply docs/source/index.rst]: I agree with @jmikovic , contents from the both links should be moved into RTD. I don't see the reason why they are buried inside leapp-repository.

File: docs/source/index.rst [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
+ - Architecture overview of `leapp and snactor tools <https://github.com/leapp-to/leapp-to.github.io/blob/master/architecture.md>`_.
```
Comment (examon) [inline docs/source/index.rst]: This link points to the archived repo. I think we should move the content directly to the RTD.

File: docs/source/index.rst [hunk 5]
```diff
@@ -1,6 +1,15 @@
 Welcome to developer documentation for Leapp!
-=================================
-   `Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework
+=============================================
+`Leapp <https://oamg.github.io/leapp/>`_ is an OS and application modernization framework.
+
+Resources not included in this RTD:
+ - RPM dependencies for `leapp-repository <https://github.com/oamg/leapp-repository/blob/master/docs/dependencies.md>`_.
+ - Upgrade inhibition by producing a `inhibiting report message <https://github.com/oamg/leapp-repository/blob/master/docs/howto-inhibit-upgrade.md>`_.
+ - Type of tests used for `testing the actors <https://github.com/oamg/leapp-repository/blob/master/docs/tests.md>`_.
```
Comment (examon) [inline docs/source/index.rst]: I think this should be moved/edited to the RTD testing page.

**Comment by vojtechsokol** [comment]
Regarding the comments that most of the linked content should be moved to RTD - I agree, I was planning to write it to a comment, but you were faster than me. Anyway, the task this PR is part of should only involve adding links with content that is not part of the RTD. There are also other issues (e.g. dependencies.md is not linked from anywhere in RTD and is accessible only by using search function), but it should be solved in separate PR.

File: docs/source/index.rst (hunk 1 above)
Comment (vojtechsokol) [reply docs/source/index.rst]: removed

File: docs/source/index.rst (hunk 3 above)
Comment (vojtechsokol) [reply docs/source/index.rst]: removed

**Pull request merged**